modules/hcp-disable-feature.adoc

```diff
@@ -5,7 +5,11 @@
 [id="hcp-disable-feature_{context}"]
 = Disabling the {hcp} feature
 
-Before you disable the {hcp} feature, you must first uninstall the HyperShift Operator. 
+To disable the {hcp} feature, complete the following steps.
```

Comment:
If you need I would prefer to read something like "You can disable the {hcp} feature to improve the performance of your cluster."